SCons/Tool/Tool.xml

```diff
@@ -37,10 +37,19 @@ Example:
 
 <example_commands>
 # builds foo.c
-env.CFile(target = 'foo.c', source = 'foo.l')
+env.CFile(target='foo.c', source='foo.l')
+
 # builds bar.c
-env.CFile(target = 'bar', source = 'bar.y')
+env.CFile(target='bar', source='bar.y')
 </example_commands>
+
+<para>
```

Comment:
Not sure we really need this notice.

Reply:
Maybe not, but it could be a little surprising to traditional yacc users. Can certainly remove.

Reply:
Made the note non-visible.